### ISSUE TITLE
idea for new features

### ISSUE THREAD

**Issue by weisseskopfadler1** [easydiffusion/easydiffusion#1720]
1. Add the function "just resize" img2img
2. add a “transform” function or something like that... To transform the picture using another model and properties (for example, to make the picture photorealistic)

**Comment by fdwr** [comment]
> Add the function "just resize" img2img

Yeah, I've often wanted a pure upsampling mode that simply took the original image, its matching prompt, and then upsampled it with more detail, but *without* modifying the original image's detail (and I mean via true SD image-to-image rather than ESGRAN or any of those others which look bad when you zoom in). However, image-to-image upsampling always changes little details. 🤔 So I've been thinking of hacking ED locally to introduce much less additional noise each iteration and to restrain any denoising so that the average of the output image pixel window equals the average of the input image pixels (e.g. if you're upscaling by 2x, then every 2x2 window in the output should match the average of a 1x1 window in the input).

**Comment by cmdr2** [comment]
Thanks! Will keep a pure upscaling option in mind in an upcoming version.

I'm not sure what the second suggestion means - you can already set an initial image, and generate images without setting a prompt (using whichever model you've set).
